cmake/projects/flatbuffers/hunter.cmake

```diff
@@ -13,19 +13,19 @@ hunter_add_version(
     PACKAGE_NAME
     flatbuffers
     VERSION
-    1.3.0-p3
+    1.4.0-p1
```

Comment:
Please add new version instead of rewriting. Old version still can be specified by custom user config: https://github.com/ruslo/hunter/wiki/example.custom.config.id#custom-package-version

Set new version as default here: https://github.com/ruslo/hunter/blob/master/cmake/configs/default.cmake